.husky/pre-commit

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+export PATH="/Users/enriquesanchez/.nvm/versions/node/v16.14.0/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin"
```

Comment:
-2 there has to be a different solution for the problems you're having with `yarn`. adding this user-specific `PATH` will cause this on every other machine
```
❯ git commit -m "test"
.husky/pre-commit: line 6: yarn: command not found
```

Reply:
Oh! I did not mean to include that change here!!! 🙈 

@BartoszJarocki Is there a way I can remove it from this PR?

Reply:
@BartoszJarocki Ok, I saw that I could revert and commit again. Let me know how all looks to you now 👍